### ISSUE TITLE
Uncaught exception in webgme client when going into any of the tabs

### ISSUE THREAD

**Issue by azeey** [dfst/deepforge-caffe#170]
is the downloaded error log:

```
{
  "timestamp": "2016-04-26T14:40:04.978Z",
  "webgme": {
    "NpmVersion": "1.7.1",
    "version": "1.7.1",
    "GitHubVersion": "webgme@>=1.5.1 <2.0.0"
  },
  "branchErrors": [],
  "browserInfo": {
    "appCodeName": "Mozilla",
    "appName": "Netscape",
    "appVersion": "5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.112 Safari/537.36",
    "onLine": true,
    "cookieEnabled": true,
    "platform": "Linux x86_64",
    "product": "Gecko",
    "userAgent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.112 Safari/537.36"
  },
  "clientState": {
    "connection": "CONNECTED",
    "renewingToken": false,
    "exception": {
      "message": "this.setReadOnly is not a function",
      "stack": "TypeError: this.setReadOnly is not a function\n    at ModelEditorControl.selectedObjectChanged (http://localhost:8080/js/Panels/ModelEditor/ModelEditorControl.js:162:22)\n    at ModelEditorControl._stateActiveObjectChanged (http://localhost:8080/js/Panels/ModelEditor/ModelEditorControl.js:991:18)\n    at b (http://localhost:8080/dist/webgme.dist.build.js:13:3868)\n    at v (http://localhost:8080/dist/webgme.dist.build.js:13:3602)\n    at h (http://localhost:8080/dist/webgme.dist.build.js:13:1521)\n    at u.trigger (http://localhost:8080/dist/webgme.dist.build.js:13:3495)\n    at http://localhost:8080/panel/AutoViz/AutoVizPanel.js:120:36\n    at Object.i.execCb (http://localhost:8080/lib/require/require.min.js:29:311)\n    at Object.$.check (http://localhost:8080/lib/require/require.min.js:18:423)\n    at Object.$.enable (http://localhost:8080/lib/require/require.min.js:23:238)"
    },
    "projectAccess": {
      "read": true,
      "write": true,
      "delete": true
    },
    "core": "instantiated",
    "branchName": "master",
    "branchStatus": "SYNC",
    "readOnlyProject": false,
    "viewer": false,
    "users": [
      "8a848b39-3cf0-d069-ade8-47f167ad32c6",
      "cfdbaae5-5441-faa1-7647-93603741490e",
      "a783cf4e-e6e0-017f-4fd8-e3439b096844",
      "91e0adb1-81b7-bc37-8313-acad35de3949",
      "0213065c-984e-ad42-ca9d-a3dfee128d05"
    ],
    "nodes": [
      "",
      "/1567574167",
      "/1",
      "/74313480",
      "/302131555",
      "/304419501",
      "/512112612",
      "/651633694",
      "/801049710",
      "/1213207378",
      "/1470060635",
      "/1833994406",
      "/1954214071",
      "/n",
      "/385167607",
      "/555527267",
      "/833202588"
    ],
    "loadNodes": [],
    "metaNodes": [
      "",
      "/1567574167",
      "/1",
      "/74313480",
      "/302131555",
      "/304419501",
      "/512112612",
      "/651633694",
      "/801049710",
      "/1213207378",
      "/1470060635",
      "/1833994406",
      "/1954214071",
      "/n",
      "/385167607",
      "/555527267",
      "/833202588"
    ],
    "rootHash": "#1b473da972f4482dee32fb6a5dee1498fc53c85a",
    "commitHash": "#32ed941c9eb666df757bddfdb30fcc3729dd3657",
    "undoRedoChain": {
      "previous": null,
      "next": null
    },
    "inTransaction": false,
    "msg": "",
    "gHash": 0,
    "loadError": null,
    "ongoingTerritoryUpdateCounter": 0,
    "ongoingLoadPatternsCounter": 0,
    "pendingTerritoryUpdatePatterns": {},
    "loadingStatus": null,
    "inLoading": false,
    "loading": {
      "rootHash": null,
      "commitHash": null
    }
  }
}
```

**Comment by brollb** [comment]
The main part of the problem seems to be
`TypeError: this.setReadOnly is not a function\n at
ModelEditorControl.selectedObjectChanged
(http://localhost:8080/js/Panels/ModelEditor/ModelEditorControl.js:162:22)\n
at ModelEditorControl._stateActiveObjectChanged`

One of the visualizers subclasses the ModelEditor from webgme and is not up
to date with the newest version of ModelEditor. Specifically, a method
called "setReadOnly" was introduced. This was add in v1.7.0 of webgme.

**Comment by brollb** [comment]
Fixed by #171
